src/main/java/dk/cs/aau/huppaal/controllers/HUPPAALController.java

```diff
@@ -216,6 +217,47 @@ public void initialize(final URL location, final ResourceBundle resources) {
         // Keybind for deleting the selected elements
         KeyboardTracker.registerKeybind(KeyboardTracker.DELETE_SELECTED, new Keybind(new KeyCodeCombination(KeyCode.DELETE), this::deleteSelectedClicked));
 
+        // Register a key-bind for showing debug-information
+        KeyboardTracker.registerKeybind("DEBUG", new Keybind(new KeyCodeCombination(KeyCode.F12), () -> {
+            // Toggle the debug mode for the debug class (will update misc. debug variables which presentations bind to)
+            Debug.debugModeEnabled.set(!Debug.debugModeEnabled.get());
+
+            if (HUPPAAL.debugStage != null) {
+                HUPPAAL.debugStage.close();
+                HUPPAAL.debugStage = null;
+                return;
+            }
+
+            try {
+                final UndoRedoHistoryPresentation undoRedoHistoryPresentation = new UndoRedoHistoryPresentation();
+                undoRedoHistoryPresentation.setMinWidth(100);
+
+                final BackgroundThreadPresentation backgroundThreadPresentation = new BackgroundThreadPresentation();
+                backgroundThreadPresentation.setMinWidth(100);
+
+                final HBox root = new HBox(undoRedoHistoryPresentation, backgroundThreadPresentation);
+                root.setStyle("-fx-background-color: brown;");
+                HBox.setHgrow(undoRedoHistoryPresentation, Priority.ALWAYS);
+                HBox.setHgrow(backgroundThreadPresentation, Priority.ALWAYS);
+
+
+                HUPPAAL.debugStage = new Stage();
+                HUPPAAL.debugStage.setScene(new Scene(root));
+
+                HUPPAAL.debugStage.getScene().getStylesheets().add("main.css");
+                HUPPAAL.debugStage.getScene().getStylesheets().add("colors.css");
+
+                var vb = Screen.getPrimary().getVisualBounds();
+                HUPPAAL.debugStage.setWidth(vb.getWidth() * 0.2);
+                HUPPAAL.debugStage.setHeight(vb.getWidth() * 0.3);
+
+                HUPPAAL.debugStage.show();
+                //stage.requestFocus();
+            } catch (final Exception e) {
+                e.printStackTrace();
+            }
+        }));
+
```

Comment:
Good idea to move this :+1:

Reply:
I cleaned it up just a bit more in a70e42e569fc52d79ef51da68e43b0319633810d